src-docs/src/views/i18n/i18n_renderprop.js

```diff
@@ -11,7 +11,12 @@ export default () => {
       </p>
       <div>
         <EuiI18n token="euiI18nRenderprop.placeholderName" default="John Doe">
-          {placeholderName => <EuiFieldText placeholder={placeholderName} />}
+          {placeholderName => (
+            <EuiFieldText
+              placeholder={placeholderName}
+              aria-label="Passing EuiI18n as a render prop child "
+            />
```

Comment:
Let's solve this with a visual label instead of an `aria-label`.

This isn't tested but something like this should work (and then you can also delete the `<p>` above):
```suggestion
		<EuiFormRow label={<>This text field&apos;s placeholder reads from <EuiCode>i18n.renderpropexample</EuiCode></>}>
            <EuiFieldText placeholder={placeholderName}/>
        </EuiFormRow>
```